app/templates/about.hbs

```diff
@@ -1,99 +1,198 @@
-    Tvým úkolem ve většině úloh bude řešení algoritmického či programátorského problému.
-    Někdy bude stačit popsat řešení jen obecně pomocí pseudokódu, jindy po tobě budeme chtít i&nbsp;program v&nbsp;Pythonu.
-    Pokud programovat neumíš nebo programuješ v&nbsp;jiném jazyce, nemusíš se ničeho bát.
-    Python tě pomocí cvičných příkladů rychle naučíme.
-    Vždyť přece kolik jazyků umíš, tolikrát jsi člověkem. ;)
-    </p>
-    <p>Některé z&nbsp;našich úloh mohou být složité nebo pracné, i&nbsp;když s&nbsp;dobrým nápadem se většinou dají vyřešit rychle a&nbsp;elegantně. Není třeba vyřešit všechny úlohy &ndash; pošli nám, co zvládneš, a&nbsp;zbytek si po uzavření úlohy můžeš přečíst ve vzorovém řešení. Také není nutné, abys měl(a) úlohu kompletně dořešenou. I&nbsp;pokud máš jen nějaké nápady co s&nbsp;ní, napiš nám je a&nbsp;my tvoji snahu určitě oceníme.</p>
-
-    <h3>Proč se účastnit?</h3>
-    <ul>
-    <li><p>Procvičíš si programátorské, matematické a&nbsp;logické myšlení a&nbsp;schopnost řešit nelehké problémy s&nbsp;využitím počítače. Získáš nové znalosti a&nbsp;naučíš se algoritmy běžně používané v&nbsp;každodenní programátorské praxi.</p></li>
-
-    <li><p>Naučíš se své myšlenky jasně formulovat a&nbsp;srozumitelně vyjadřovat.</p></li>
-
-    <li><p>Připravíš se na programátorské i&nbsp;jiné soutěže, např. na matematickou olympiádu kategorie&nbsp;P, ale také na budoucí přijímací zkoušky či pohovory. Trénink je základ úspěchu!</p></li>
-
-    <li><p>Přiblíží se ti látka vyučovaná na vysokých školách. Dokážeš se lépe rozhodnout, co chceš studovat, a&nbsp;pokud se rozhodneš pro informatiku, získáš mírný náskok na své budoucí spolužáky.</p></li>
-
-    <li><p>Budeš-li patřit mezi úspěšné řešitele, můžeš se těšit na přijetí na Fakultu informatiky MU bez přijímacích zkoušek.</p></li>
-
-    <li><p>A&nbsp;hlavně, když budeš svědomitě řešit, jistě tě nemine odměna v&nbsp;podobě <a href="https://kscuk.fi.muni.cz/">týdenního soustředění</a>.</p></li>
-
-    </ul>
-
-  <h2 id="faq">FAQ</h2>
-
-    <div class="panel panel-ksi">
-      <div class="panel-heading panel-heading-ksi">Q: Som Slovák. Môžem písať riešenie v&nbsp;slovenčine?</div>
-      <div class="panel-body">
-        A: Jistě. V&nbsp;organizátorském týmu máme dokonce několik Slováků, takže se slovenština občas objeví i&nbsp;v&nbsp;zadání.
-      </div>
+     Pokud používáš při řešení nějaký externí zdroj (literaturu, webovou
+     stránku, konzultaci s kamarádem, apod.) a čerpáš odtud nějakou klíčovou
+     myšlenku, nebo třeba kus textu, musíš tento zdroj uvést formou odkazu, na
+     základě kterého budeme moct tento zdroj dohledat.
+    </p></li>
+   </ol>
+
+ <h2 id="calendar">Kalendář akcí KSI</h2>
+  <p>
+  Významné akce KSI, jako Ksílety nebo termíny odevzdání úloh, zaznamenáváme do
+  <a
+  href="https://calendar.google.com/calendar/embed?src=07cn4qi2qm1g3rtpuls78dmsbg%40group.calendar.google.com"
+  target="_blank">Google kalendáře KSI</a>. Tento kalendář si můžeš <a

```

Comment:
Mít na řádku pouze začátek link tagu a jinak nic je přinejmenší kontroverzní.

Reply:
Klidně to spoj, je mi to jedno.